apps/system/test/unit/accessibility_test.js

```diff
@@ -289,4 +313,67 @@ suite('system/Accessibility', function() {
         accessibility.quicknav.element.classList.contains('visible'));
     });
   });
+
+  suite('interaction hints', function() {
+    var screenNode, clock;
+    var realL10n = navigator.mozL10n;
+
+    setup(function() {
+      screenNode = document.createElement('div');
+      screenNode.id = 'screen';
+      document.body.appendChild(screenNode);
+
+      clock = this.sinon.useFakeTimers();
+
+      navigator.mozL10n = MockL10n; // jshint ignore:line
+    });
+
+    teardown(function() {
+      screenNode.parentNode.removeChild(screenNode);
+      clock.restore();
+      navigator.mozL10n = realL10n;
+    });
+
+    test('handle vc change event with hints enabled', function() {
+      var stubSetHintsTimeout = this.sinon.stub(accessibility,
+        'setHintsTimeout');
+      // This is where MozL10n was necessary in place of the stubbed
+      // accessibility.speak method
+      accessibility.handleAccessFuOutput(vcChangeHintsEnabledDetails);
+      clock.tick(2000);
```

Comment:
No need to tick here, we moch SpeechSynthesis to be synchronous. see https://github.com/mozilla-b2g/gaia/blob/master/apps/system/test/unit/mock_speech_synthesis.js